spec/poros/quote_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Quote do
+  it 'exists' do
+    quote_data = {
```

Comment:
I like that you are passing just the data you need and not making a full API call just to test the poro. This is great practice. You _could_ even pare this down further and pass in only the key:value pairs of the data you need, but this is fine too.